b2share/config.py

```diff
@@ -490,3 +490,10 @@
 #: There is no password so don't send password change emails
 SECURITY_SEND_PASSWORD_CHANGE_EMAIL=False
 SECURITY_SEND_PASSWORD_RESET_NOTICE_EMAIL=False
+
+if os.environ.get("B2ACCESS_CONSUMER_KEY") and os.environ.get("B2ACCESS_SECRET_KEY") and os.environ.get("B2SHARE_CONFIG_ENABLE_B2ACCESS",0) == 0:
```

Comment:
Shouldn't the last part be `== 1`? And this can be removed completely from config.py because of configloader.